api/src/main/java/data/Drills/Drill.java

```diff
@@ -0,0 +1,71 @@
+package data.Drills;
+
+import java.util.List;
+
+/**
+ * Created by Michaela Bamburová on 13.05.2016.
+ */
+public class Drill {
+
+	private final String drillId;
+	private final String subject;
+	private final String description;
+	private final List<Question> questions;
+
+
+	public Drill(String drillId, String subject, String description, List<Question> questions) {
+		this.drillId = drillId;
```

Comment:
V rámci tohoto PR by to chtělo najít nějakou používanou libku na generování UUID v4 a kdyby měla statickej generátor tak by to bylo úplně nejlepší - dalo by se to použít pěkně v konstruktoru.